app/views/books/_addable_field.html.erb

```diff
@@ -13,4 +13,4 @@
         </div>
       <% end %>
 <% end %>
-<%= button_tag "New #{singular.capitalize}", :type => 'button', id: "add_#{singular}"%>
\ No newline at end of file
+<%= button_tag button_name, :type => 'button', id: "add_#{singular}"%>
```

Comment:
I like the idea of this partial a lot. 
What is `button_name`

Reply:
New Author or New Category